src/utils/coordinates/coordinateExtractors.js

```diff
@@ -75,28 +73,119 @@ const REGEX_WGS_84_WITH_SECONDS_PREFIXED = new RegExp(
     'i'
 )
 
+/**
+ * @param {String} text
+ * @returns {[Number, Number] | undefined}
+ */
+function extractWGS84Coordinates(text) {
+    const regexMatch = [
+        REGEX_WGS_84,
+        REGEX_WGS_84_WITH_CARDINALS,
+        REGEX_WGS_84_WITH_PRE_FIXED_CARDINALS,
+        REGEX_WGS_84_WITH_MIN,
+        REGEX_WGS_84_WITH_MIN_PREFIXED,
+        REGEX_WGS_84_WITH_SECONDS,
+        REGEX_WGS_84_WITH_SECONDS_PREFIXED,
+    ]
+        .map((regex) => regex.exec(text.trim()))
+        .find((result) => Array.isArray(result))
+    if (regexMatch) {
+        return wgs84Extractor(regexMatch)
+    }
+    return undefined
+}
+
 // LV95, LV03, metric WebMercator (EPSG:3857)
 const REGEX_METRIC_COORDINATES =
     /^(?<coord1>\d{1,3}(['`´ ]?\d{3})*(\.\d+)?)\s*[,/ \t]\s*(?<coord2>\d{1,3}(['`´ ]?\d{3})*(\.\d+)?)$/i
 
+/**
+ * @param {String} text
+ * @returns {[Number, Number] | undefined}
+ */
+function extractLV95Coordinates(text) {
+    const coordinates = numericalExtractor(REGEX_METRIC_COORDINATES.exec(text.trim()))
+    if (coordinates) {
+        if (LV95.isInBounds(coordinates[0], coordinates[1])) {
+            return [coordinates[0], coordinates[1]].map(LV95.roundCoordinateValue)
+        } else if (LV95.isInBounds(coordinates[1], coordinates[0])) {
+            return [coordinates[1], coordinates[0]].map(LV95.roundCoordinateValue)
+        }
+    }
+    return undefined
+}
+
+/**
+ * @param {String} text
+ * @returns {Promise<[Number, Number] | undefined>}
+ */
+async function extractLV03Coordinates(text) {
+    const coordinates = numericalExtractor(REGEX_METRIC_COORDINATES.exec(text.trim()))
+    if (coordinates) {
+        if (LV03.isInBounds(coordinates[0], coordinates[1])) {
+            return reframe({
+                inputCoordinates: [coordinates[0], coordinates[1]],
+                inputProjection: LV03,
+            })
+        } else if (LV03.isInBounds(coordinates[1], coordinates[0])) {
+            return reframe({
+                inputCoordinates: [coordinates[1], coordinates[0]],
+                inputProjection: LV03,
+            })
+        }
+    }
+    return undefined
+}
```

Comment:
See other comment I would not do the reframe here and keep the function as the other projection, doing only one thing, extracting. This would make the code easier to understand/read.

Reply:
The return value should be in correct order x,y like for lv95 this check can be done here to normalize the coordinate axis order.